API-Samples/Java/app/src/main/java/videoindexersamples/VideoIndexerClient.java

```diff
@@ -0,0 +1,239 @@
+    private static final String AzureResourceManager = "https://management.azure.com";
+    private static final String SubscriptionId = "<Your_Subscription_Id_here>";
+    private static final String ResourceGroup = "<Your_Resource_Group_here>";
+    private static final String AccountName = "<Your_Account_Name_Here>";
+    private static final String ApiVersion = "2022-08-01";
+    private static final String ApiUrl = "https://api.videoindexer.ai";
+    private final String armAccessToken;
+    private final Gson gson;
+    private String accountAccessToken;
+    private Account account = null;
+
+    private VideoIndexerClient(String armAccessToken) {
+        this.armAccessToken = armAccessToken;
+        gson = new GsonBuilder().setPrettyPrinting().create();
+    }
+
+    /**
+     * Create a new Video Indexer Client.
+     * During Creation a DefaultAzureCredential Chain is being invoked .
+     *
+     * @return the Video Indexer Client
+     */
+    public static VideoIndexerClient create() {
+        var tokenRequestContext = new TokenRequestContext();
+        tokenRequestContext.addScopes(String.format("%s/.default", AzureResourceManager));
+
+        DefaultAzureCredential defaultCredential = new DefaultAzureCredentialBuilder().build();
+        String accessToken = Objects.requireNonNull(defaultCredential.getToken(tokenRequestContext).block()).getToken();
+        return new VideoIndexerClient(accessToken);
+    }
+
+    /**
+     * @param permission - the required permission on the Video Indexer Account
+     * @param scope      - The scope of the Access token
+     * @param videoId    - the Video ID
+     * @param projectId  - the Project ID
+     */
+    public void getAccountAccessToken(ArmAccessTokenPermission permission, ArmAccessTokenScope scope, String videoId, String projectId) {
+        var accessTokenRequest = new AccessTokenRequest(projectId, videoId, permission, scope);
+        var accessTokenRequestStr = gson.toJson(accessTokenRequest);
+
+        var requestUri = String.format("%s/subscriptions/%s/resourcegroups/%s/providers/Microsoft.VideoIndexer/accounts/%s/generateAccessToken?api-version=%s", AzureResourceManager, SubscriptionId, ResourceGroup, AccountName, ApiVersion);
+        try {
+            HttpRequest request = HttpRequest.newBuilder()
+                    .uri(new URI(requestUri))
+                    .headers("Content-Type", "application/json;charset=UTF-8")
+                    .headers("Authorization", String.format("Bearer %s", this.armAccessToken))
+                    .POST(HttpRequest.BodyPublishers.ofString(accessTokenRequestStr))
+                    .build();
+
+            var response = httpStringResponse(request);
+            AccessTokenResponse accessTokenResponse = gson.fromJson(response.body(), AccessTokenResponse.class);
+            this.accountAccessToken = accessTokenResponse.accessToken;
+        } catch (URISyntaxException | IOException | InterruptedException e) {

```

Comment:
ex instead of e

Reply:
actuaully that was IntelliJ auto-generated code , but I have changed it to ex